node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -147,6 +170,90 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
   def isDisconnected(idx: Int): Future[Boolean] =
     peerMsgSenders(idx).isDisconnected()
 
+  private def createInDbIfBlockFilterPeer(peer: Peer): Future[Unit] = {
+    if (peerData(peer).serviceIdentifier.nodeCompactFilters) {
+      logger.info(s"Our peer=$peer has been initialized")
+      PeerDAO().upsertPeer(
+        s"${peer.socket.getHostString}:${peer.socket.getPort}")
+    } else {
+      logger.info(
+        s"Our peer=$peer does not support compact filters. Disconnecting.")
+      peerData(peer).peerMessageSender.disconnect()
+    }
+    Future.unit
+  }
+
+  def createInDbIfBlockFilterPeer(networkAddress: NetworkIpAddress): Unit = {
+    val ipv4bytes: Option[ByteVector] =
+      try {
+        Some(networkAddress.address.ipv4Bytes)
+      } catch {
+        case _: Throwable => {
+          logger.info("Ignoring ipv6 address from addr message")
```

Comment:
why do we ignore ipv6 addresses?

Reply:
also nit: don't need braces around this

Reply:
For some context of conversations I had with Shreyansh, i think we decided to limit scope and try to get something working to start, and then add other address types later on. 

Do you have differing opinions on this? Perhaps you and Shreyansh could colloborate how to best build the data model for this.

Reply:
Let me take a stab at it and see if my idea will work